### PR TITLE
General interrim parser changes

### PR DESCRIPTION
**Pull request by adrianparvino**
Fixes https://github.com/lapp0/nix-gui/issues/14
Fixes https://github.com/lapp0/nix-gui/issues/17

### CODE REVIEW

File: nixui/parser/ast.py [hunk 1]
```diff
@@ -100,9 +100,13 @@ def visit_node(self, node, visited_children):
     def visit_token(self, node, visited_children):
         """ Returns the overall output. """
         name, _, quoted, _, _, position, _ = visited_children
-        quoted = quoted[1:-1]  # strip quote symbols (TODO: move this to the grammar)
-        quoted = quoted.replace('EQ', '\"')  # TODO: fix hack (see note 3929 above)
-        quoted = quoted.encode('utf-8').decode('unicode_escape')  # unescape
+        quoted = quoted\
```
Comment (lapp0) [inline nixui/parser/ast.py:103]: Can pythons standard library perform this multi-line operation in a single line?

File: 0001-Interrim-solution-for-Unicode-support.patch [hunk 2]
```diff
@@ -0,0 +1,39 @@
+From 4e2578b1302a098363e6fca7217a9f368b57d727 Mon Sep 17 00:00:00 2001
```
Comment (lapp0) [inline 0001-Interrim-solution-for-Unicode-support.patch:1]: Looks good, should this patch be submitted upstream though?

Comment (aaronjanse) [reply 0001-Interrim-solution-for-Unicode-support.patch:1]: An upstream PR would be great!

File: nixui/parser/ast.py (hunk 1 above)
Comment (adrianparvino) [reply nixui/parser/ast.py:103]: TIL this exists: https://docs.python.org/3/library/stdtypes.html#str.translate

I'll replace it with that.

Comment (adrianparvino) [reply nixui/parser/ast.py:103]: This was a dumb from me lol. `translate` is a translation from character to character(or character to string). What we need is a translation from string to character.

**Comment by adrianparvino** [comment]
Superseded by https://github.com/nix-gui/nix-gui/pull/42

**Pull request closed without merging**